### PR TITLE
Update memcached minio nodejs linting fixes

### PR DESCRIPTION
**Pull request by mshahzeb**
Linting fixes and updates for mixinification of integrations [here](https://github.com/grafana/cloud-onboarding/pull/2837).

### CODE REVIEW

File: memcached-mixin/alerts.libsonnet [hunk 1]
```diff
@@ -17,6 +17,8 @@
               message: |||
                 Memcached Instance {{ $labels.job }} / {{ $labels.instance }} is down for more than 15mins.
               |||,
+              description: 'Memcached Instance {{ $labels.job }} / {{ $labels.instance }} is down for more than 15mins.',
```
Comment (v-zhuravlev) [inline memcached-mixin/alerts.libsonnet:20]: I think that message should be removed if we have description and summary instead.

Comment (v-zhuravlev) [inline memcached-mixin/alerts.libsonnet:20]:
```suggestion
              description: 'Memcached Instance {{ $labels.job }} / {{ $labels.instance }} is down for more than 15 minutes.',
```

File: memcached-mixin/alerts.libsonnet [hunk 2]
```diff
@@ -47,6 +51,8 @@
               message: |||
                 Memcached Instance {{ $labels.job }} / {{ $labels.instance }} connection usage is at {{ printf "%0.0f" $value }}% for at least 15m.
               |||,
+              description: 'Memcached Instance {{ $labels.job }} / {{ $labels.instance }} connection usage is at {{ printf "%0.0f" $value }}% for at least 15m.',
```
Comment (v-zhuravlev) [inline memcached-mixin/alerts.libsonnet:54]:
```suggestion
              description: 'Memcached Instance {{ $labels.job }} / {{ $labels.instance }} connection usage is at {{ printf "%0.0f" $value }}% for at least 15 minutes.',
```

**Review by v-zhuravlev — APPROVED** [review]
Would do you think about capitalising alerts?
Like
minioDisksOffline -> MinioDisksOffline
https://github.com/grafana/jsonnet-libs/pull/882/files#diff-d76470bfc803b64abe64c42a2805143108c0abd211c697e658604915df07ac5bR7


"The alert name is a terse description of the alerting condition, using camel case, without whitespace, starting with a capital letter. "
https://monitoring.mixins.dev/#guidelines-for-alert-names-labels-and-annotations

File: memcached-mixin/alerts.libsonnet (hunk 1 above)
Comment (Dasomeone) [reply memcached-mixin/alerts.libsonnet:20]: +1, especially as the linter ignores 'message', so the format could end up wonky

File: minio-mixin/.lint [hunk 3]
```diff
@@ -0,0 +1,5 @@
+exclusions:
+  panel-units-rule:
+    reason: "The dashboard already has units but it is using older panel versions for which this rule does not work correctly"
+  alert-name-camelcase:
+    reason: "Rule incorrectly giving warning while alert names are already in camelCase"
```
Comment (Dasomeone) [inline minio-mixin/.lint:5]: Should specify that the alert rule is due to having abbreviations/acronyms in the title, as info for future maintainers

**Review by Dasomeone — CHANGES_REQUESTED** [review]

File: memcached-mixin/alerts.libsonnet (hunk 1 above)
Comment (mshahzeb) [reply memcached-mixin/alerts.libsonnet]: Thanks! Removed messages

File: minio-mixin/.lint (hunk 3 above)
Comment (mshahzeb) [reply minio-mixin/.lint]: I have capitalized the initial letters and this rule is not giving an issue more so removed the lint exclusion.

**Review by rgeyer — APPROVED** [review]
🚀

**Review by Dasomeone — APPROVED** [review]

**Pull request merged**